plugins/hep-python-tools/VENDORED-LICENSES.md

```
@@ -0,0 +1,45 @@
+
+The following skills were vendored from the
+[IRIS-HEP marketplace](https://github.com/iris-hep/iris-hep-marketplace):
+
+- `skills/cli-creator/`
+- `skills/standalone-script/`
+
+**Original authors:** Gordon Watts, Ben Galewsky
+
+**License:** BSD 3-Clause License
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)